foca/database/register_mongodb.py

```diff
@@ -63,47 +60,41 @@ def register_mongodb(
 
 def create_mongo_client(
         app: Flask,
-        host: str = 'mongodb',
-        port: int = 27017,
-        db: str = 'database',
+        settings: MongoConfig
 ) -> PyMongo:
     """Register MongoDB database with Flask app.
 
     Optionally, basic authorization can be set by environment variables.
 
     Args:
         app: Flask application object.
-        host: Host at which the MongoDB database is exposed.
-        port: Port at which the MongoDB database is exposed.
-        db: Name of the database to be accessed/created.
+        settings: An instance of the MongoConfig class
 
     Returns:
-        Client for the MongoDB database specified by `host`, `port` and `db`.
+        Client for the MongoDB database specified by `host`, `port` and `db`
+        attributes of the MongoConfig instance.
     """
-    if os.environ.get('MONGO_USERNAME') != '':
-        auth = '{username}:{password}@'.format(
-            username=os.environ.get('MONGO_USERNAME'),
-            password=os.environ.get('MONGO_PASSWORD'),
+
+    if settings.username:
+        settings.auth = '{username}:{password}@'.format(
```

Comment:
While you're at it, might as well refactor to use f-strings as in most other places. Much more concise and easier to read than `.format()`